accounts.go

```diff
@@ -40,9 +40,13 @@ type Account struct {
 
 type GetAccountInput struct{}
 
-func (client *AccountsClient) GetAccount(input *GetAccountInput) (*Account, error) {
+func (client *AccountsClient) GetAccount(ctx context.Context, input *GetAccountInput) (*Account, error) {
+	if ctx == nil {
+		panic(nilContext)
```

Comment:
The idiomatic approach to things requiring a context is to panic if it is nil - we should probably roll that change through the entire API though (or delegate it to the client, though that will lead to worse backtraces).